toqito/channel_metrics/completely_bounded_spectral_norm.py

```diff
@@ -32,4 +32,5 @@ def completely_bounded_spectral_norm(phi: np.ndarray) -> float:
     :return: The completely bounded spectral norm of the channel
 
     """
-    return completely_bounded_trace_norm(dual_channel(phi))
+    dual_as_array = np.asarray(dual_channel(phi))
+    return completely_bounded_trace_norm.completely_bounded_trace_norm(dual_as_array)
```

Comment:
Don't know if this is a best practice hack. 

Due to `dual_channel`, the return type of `dual_channel(phi)` is ` np.ndarray | List[List[np.ndarray]]` when `completely_bounded_trace_norm` expects just a `np.ndarray`. So, added an additional step of converting `dual_channel(phi)`. 

If `completely_bounded_trace_norm.completely_bounded_trace_norm` was not specified, mypy failed with `toqito/channel_metrics/completely_bounded_spectral_norm.py:35: error: Module not callable  [operator]`